### PR TITLE
 replace count() usage to empty() when possible

### PR DESCRIPTION
**Pull request by samsonasik**
ref: http://maettig.com/code/php/php-performance-benchmarks.php#empty

### CODE REVIEW

File: src/Helper/HeadLink.php [hunk 1]
```diff
@@ -387,15 +387,15 @@ public function createDataStylesheet(array $args)
             return false;
         }
 
-        if (0 < count($args)) {
+        if (! empty($args)) {
```
Comment (Ocramius) [inline src/Helper/HeadLink.php]: All these `! empty($args)` can be simplified to `$args` in the context of boolean expressions

File: src/Helper/HeadLink.php [hunk 2]
```diff
@@ -387,15 +387,15 @@ public function createDataStylesheet(array $args)
...
             $media = array_shift($args);
             if (is_array($media)) {
                 $media = implode(',', $media);
             } else {
                 $media = (string) $media;
             }
         }
-        if (0 < count($args)) {
+        if (! empty($args)) {
```
Comment (Ocramius) [inline src/Helper/HeadLink.php]: same

File: src/Helper/HeadLink.php [hunk 3]
```diff
@@ -404,7 +404,7 @@ public function createDataStylesheet(array $args)
             }
         }
 
-        if (0 < count($args) && is_array($args[0])) {
+        if (! empty($args) && is_array($args[0])) {
```
Comment (Ocramius) [inline src/Helper/HeadLink.php]: same

File: src/Helper/HeadLink.php [hunk 4]
```diff
@@ -452,7 +452,7 @@ public function createDataAlternate(array $args)
         $type  = array_shift($args);
         $title = array_shift($args);
 
-        if (0 < count($args) && is_array($args[0])) {
+        if (! empty($args) && is_array($args[0])) {
```
Comment (Ocramius) [inline src/Helper/HeadLink.php]: same

File: src/Model/ViewModel.php [hunk 5]
```diff
@@ -365,7 +365,7 @@ public function getChildren()
      */
     public function hasChildren()
     {
-        return (0 < count($this->children));
+        return (! empty($this->children));
```
Comment (Ocramius) [inline src/Model/ViewModel.php]: `return (bool) $this->children;`

**Comment by samsonasik** [comment]
@Ocramius done ;)

**Comment by Ocramius** [comment]
LGTM :+1:

**Pull request merged**